frontend/interactEM/src/contexts/nats/operatorstatus.tsx

```diff
@@ -0,0 +1,45 @@
+import { type ReactNode, createContext, useContext, useMemo } from "react"
+import { BUCKET_STATUS, OPERATORS } from "../../constants/nats"
+import { useBucketWatch } from "../../hooks/nats/useBucketWatch"
+import type { OperatorVal } from "../../types/gen"
+import { OperatorValSchema } from "../../types/operator"
+
+interface OperatorStatusContextType {
+  operators: OperatorVal[]
+  operatorsLoading: boolean
+  operatorsError: string | null
+}
+
+const OperatorStatusContext = createContext<OperatorStatusContextType>({
+  operators: [],
+  operatorsLoading: true,
+  operatorsError: null,
+})
+
+export const OperatorStatusProvider: React.FC<{ children: ReactNode }> = ({
+  children,
+}) => {
+  const {
+    items: operators,
+    isLoading: operatorsLoading,
+    error: operatorsError,
```

Comment:
Then to be consistent with the other providers, these should have the `operator` prefix